core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/urlviz/UrlVizOpPartialExec.scala

```diff
@@ -0,0 +1,44 @@
+package edu.uci.ics.texera.workflow.operators.visualization.urlviz
+
+import edu.uci.ics.amber.engine.architecture.worker.PauseManager
+import edu.uci.ics.amber.engine.common.InputExhausted
+import edu.uci.ics.amber.engine.common.rpc.AsyncRPCClient
+import edu.uci.ics.texera.workflow.common.operators.OperatorExecutor
+import edu.uci.ics.texera.workflow.common.tuple.Tuple
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, OperatorSchemaInfo}
+
+/**
+  * HTML Visualization operator to render any given HTML code
+  */
+class UrlVizOpPartialExec(
+    htmlContentAttrName: String,
+    operatorSchemaInfo: OperatorSchemaInfo
+) extends OperatorExecutor {
+
+  override def open(): Unit = {}
+
+  override def close(): Unit = {}
+
+  override def processTexeraTuple(
+      tuple: Either[Tuple, InputExhausted],
+      input: Int,
+      pauseManager: PauseManager,
+      asyncRPCClient: AsyncRPCClient
+  ): Iterator[Tuple] =
+    tuple match {
+      case Left(t) =>
+        val iframe =
+          "<!DOCTYPE html>\n<html lang=\"en\"><body><div class=\"modal-body\">\n<iframe src=\"" + t
+            .getField(
+              htmlContentAttrName
+            ) + "\" width=\"100%\" height=\"500px\"></iframe>\n</div></body>\n</html>"
```

Comment:
height hard coded to 500px may be a bad design. can we use the full size of the pop up window's size? so basically use the parent's height.